resources/crds/ripsaw_v1alpha1_flent_cr.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
+kind: Benchmark
+metadata:
+  name: flent-benchmark
+  namespace: my-ripsaw
+spec:
+  clustername: myk8scluster
+  #elasticsearch:
+  #   server: elk.server.com
+  #   port: 9200
+  #test_user: username_to_attach_to_metadata
+  workload:
+    # cleanup: true
+    name: flent
+    args:
+      image: quay.io/joconnel/flent:latest
```

Comment:
I assume the goal pre-merge would be to get rid of this and merge the flent image into the cloudbulldozer quay repo

Reply:
@jaredoconnell might want to change this to quay.io/loud-bulldozer/flent:latest as that image will get built as soon as your PR in benchmark-wrapper is merged.